modules/discover_granules_lambda/main.tf

```diff
@@ -12,7 +12,8 @@ resource "aws_lambda_function" "discover_granules" {
    runtime = "python3.8"
    filename = "${path.module}/../../package.zip"
    role = var.cumulus_lambda_role_arn
-   timeout = var.timeout
+   timeout = 900
```

Comment:
Maybe instead of hardcoding these, we can set them as passable variables into the dG module with the defaults as 900 and 512?